node/buildutils.js

```diff
@@ -33,7 +33,8 @@ exports.getExternals = function () {
 
     // download the same version of node used by the agent
     // and add node to the PATH
-    var nodeUrl = 'https://nodejs.org/dist';
+    var nodeUrl = process.env['TASK_NODE_URL'] || 'https://nodejs.org/dist';
+    nodeUrl = nodeUrl.replace(/\/$/, '');  // ensure there is no trailing slash on the base URL
```

Comment:
Could you please add null check for nodeUrl?

Reply:
Hi @anatolybolshakov - As far as I can see it is impossible for `nodeUrl` to be null here. 

Null is already checked here:

`var nodeUrl = process.env['TASK_NODE_URL'] || 'https://nodejs.org/dist';`

If the environment variable is [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) (which includes null) then it will default to  `https://nodejs.org/dist`. This happens due to the [behavior](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR) of the `||` operator

Am I missing something?

Reply:
You're right, sorry 😃 It's fine with default value